lib/cli.js

```diff
@@ -19,10 +19,7 @@ module.exports = function(program) {
     var checkerPromise;
     var defer = Vow.defer();
     var promise = defer.promise();
-    var checker = new Checker({
-        verbose: program.verbose,
-        esnext: program.esnext
```

Comment:
Didn't see where you readd it to checker. Suggesting that it wasn't used in code.